modules/manager/metadata/metadata.go

```diff
@@ -64,6 +88,6 @@ func Metadata() Info {
 		Commit:       commit,
 		ProjectName:  projectName,
 		Organization: organization,
-		Flavor:       flavor,
+		Flavor:       BuildFlavor(flavor),
```

Comment:
Could we change the above 
```go
	// Flavor is the flavor of the build.
	flavor = "oss"
```
to
```go
	// Flavor is the flavor of the build.
	flavor = OSSFlavor
```
to avoid duplication of this magic string?

Reply:
😅 sure I overlooked this, thx